alpha-build-machinery/make/examples/multiple-binaries/Makefile.test

```diff
@@ -51,3 +51,7 @@ test-rpm:
 	[[ ! -d ./_output/ ]]
 	$(MAKE) clean
 .PHONY: test-rpm
+
+test-update-codegen:
+	$(MAKE) update-codegen
```

Comment:
This isn't testing the verify target. I guess they are best tested together so lets make target `test-codegen` - I imagine the flow would be like:
1. update
2. verify && success
3. make a change in generated files
4. verify && failure
5. make update
6. verify && success

Reply:
I think I added this right, please check @tnozicka. Making the change to the generated CRD works best I think by adding a small yaml merge patch and merging it with `yq` (which should already be downloaded from the previous calls to update/verify). Just writing garbage to the file gave some weird results with subsequent calls to yq, probably due to invalid YAML.